File: MPI/mpicheck/tests/functions/1.0/MPI_Wait.f

        
        program main
        include 'mpif.h'
        
        INTEGER var_0
       INTEGER var_1(10)
       INTEGER var_2
        call mpi_wait(var_0, var_1, var_2)
       call pmpi_wait(var_0, var_1, var_2)
        end program main
    

File: MPI/mpicheck/tests/functions/1.0/MPI_Wait.c
#include <mpi.h>
int main(char argc, char**argv)
{
    /* vars */
    MPI_Request *var_0;
    MPI_Status *var_1;
    int ret;
    /* calls */
    ret = MPI_Wait(var_0, var_1);
    ret = PMPI_Wait(var_0, var_1);
    return 0;
}
